sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
+	"Resources": {
+		"ServiceRoot": {
+			"PropertyRequirements": {
+				"Chassis": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ChassisCollection"]
+				},
+				"Managers": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ManagerCollection"]
+				},
+				"SessionService": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["SessionService"]
+				},
+				"Id": {
```

Comment:
I would remove "Id"; this is already mandatory per the standard.

Reply:
Hey @mraineri  Can we have one call to be on the same page ?

We are framing the profile based on sushy. Now It's creating a confusion for us what is to be added as mandatory and what's not. For Interop profiles, there are some params to be added as Mandatory, recommened etc based on it's readRequirement and writeRequirement. It would be good to discuss these things in call so that we will bridge the gap between Sushy and RedFish standards.

Reply:
Sure, but one thing to keep in mind is a profile specifies requirements that go beyond what's in the standard. So, if something is already mandatory per the standard, there's no need to redefine it in a profile (it just adds extra cruft to track in the profile).

Reply:
got it.